project/Extra.scala

```diff
@@ -8,11 +8,26 @@ import bintray.BintrayKeys.{bintrayRepository, bintrayOrganization}
 
 object Extra {
 
-  val sbtPluginSettings = ScriptedPlugin.scriptedSettings ++ Seq(
+  val sbtPluginSettings = ScriptedPlugin.scriptedSettings.filterNot(
+      _.key.key.label == libraryDependencies.key.label) ++ Seq(
+      // https://github.com/sbt/sbt/issues/3325
+      libraryDependencies ++= {
+      CrossVersion.binarySbtVersion(scriptedSbt.value) match {
```

Comment:
https://github.com/scala-native/sbt-crossproject/blob/447264df1df049d05f83266eac07e1200b458c6d/bin/scalafmt#L13

> $HERE/.coursier bootstrap com.geirsson:scalafmt-cli_2.11:0.4.7 --main org.scalafmt.cli.Cli -o $HERE/.scalafmt

Should we use latest version of scalafmt?